gui/common_qt_functions.py

```diff
@@ -264,19 +264,20 @@ def updateSettingsFromUIChoices(self):
         # Each individual widget will need to override this function
         pass
 
-    def getCurrentDir(self):
-        defaultFuseDir = 'Cloudfuse'
+    def getWorkingDir(self):
         if platform == "win32":
+            defaultFuseDir = 'Cloudfuse'
             userDir = os.getenv('APPDATA')
-            currentDir = os.path.join(userDir, defaultFuseDir)
         else:
-            currentDir = os.getcwd()
+            defaultFuseDir = '.Cloudfuse'
+            userDir = os.getenv('HOME')
+        currentDir = os.path.join(userDir, defaultFuseDir)
```

Comment:
Looks like a few more variable names could change to fit with the `currentDir` -> `workingDir` change.

Reply:
Thanks for finding those, I will change them